canary_tests.py

```diff
@@ -144,4 +146,4 @@ async def test_monitor_is_online(self):
 
 
 if __name__ == "__main__":
-    unittest.main()
+    unittest.main()
```

Comment:
Missing newline at the end of the file.

See the bottom of https://thoughtbot.com/blog/no-newline-at-end-of-file for tips on how to configure your editor/IDE to leave it in by default.